lib/react.dart

```diff
@@ -15,7 +16,23 @@ typedef ReactComponentFactoryProxy ComponentRegistrar(
 /// Top-level ReactJS [Component class](https://facebook.github.io/react/docs/react-component.html)
 /// which provides the [ReactJS Component API](https://facebook.github.io/react/docs/react-component.html#reference)
 abstract class Component {
-  Map _context;
```

Comment:
@kealjones-wk @greglittlefield-wf isn't the plan was to implement all new features in the `ComponentV2` class?

Reply:
The old context api exposed in react-dart wasn't all that great to being with on the reactjs side it was bad and the exposed api on the dart side didn't match reactjs's either. Also I think @greglittlefield-wf and I looked into it and found no one using it. I feel that the advantages of the new context API far outweigh the old and think that it will have much better real world application in both `Component` and `ComponentV2` that i dont belive keeping the old context in `Component` helps anyone.

Reply:
We aren’t going to release any of this until 5.0.0 anyway - so there is no point putting it in a class that will be deprecated when that happens (`Component`).

Reply:
Good point, i added a note to the PR description to that effect. I still would like an implementation review on this by anyone willing so that when we do decide to implement it in `ComponentV2` we can know this approach will work.

Reply:
👍 👍